.github/workflows/android-appcenter.yml

```diff
@@ -55,11 +56,11 @@ jobs:
       run: ./gradlew assembleAndroidTest
 
     - name: Run tests on AppCenter
-      run: appcenter test run espresso 
+      run: appcenter test run espresso
         --token ${{secrets.APPCENTERAPITOKEN}}
         --app "Picovoice/Cheetah-Android"
-        --devices "Picovoice/single-android" 
+        --devices "Picovoice/single-android"
```

Comment:
Is there a reason we don't use `--devices "Picovoice/android-min-max"`?

Reply:
Not that I know of, I'll make the change